internal/handlers/flags.go

```diff
@@ -0,0 +1,378 @@
+		return
+	}
+
+	w.WriteHeader(http.StatusOK)
+	_, _ = w.Write([]byte(responseJson))
+}
+
+func SetIntValue(w http.ResponseWriter, r *http.Request) {
+	flagKey := r.PathValue("flagKey")
+
+	ctx := r.Context()
+	ctx_storage := ctx.Value(config.KeyVariable)
+	storageType := ctx_storage.(*config.FlagStorageType)
+
+	// define custom type
+	type Input struct {
+		FlagValue int64 `json:"value"`
+	}
+	var input Input
+
+	err := json.NewDecoder(r.Body).Decode(&input)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	storageType.SetInt(flagKey, input.FlagValue)
+
+	w.Header().Set("content-type", "application/json")
+	w.WriteHeader(http.StatusNoContent)
+	_, _ = w.Write([]byte(""))
+}
+
+func GetBoolValue(w http.ResponseWriter, r *http.Request) {
+	flagKey := r.PathValue("flagKey")
+
+	ctx := r.Context()
+	ctx_storage := ctx.Value(config.KeyVariable)
+	storageType := ctx_storage.(*config.FlagStorageType)
+	value, err := storageType.GetBool(flagKey)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusNotFound)
+		return
+	}
+
+	responseJson, err := json.Marshal(responseType{Value: value})
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	w.WriteHeader(http.StatusOK)
+	_, _ = w.Write([]byte(responseJson))
+}
+
+func SetBoolValue(w http.ResponseWriter, r *http.Request) {
+	flagKey := r.PathValue("flagKey")
+
+	ctx := r.Context()
+	ctx_storage := ctx.Value(config.KeyVariable)
+	storageType := ctx_storage.(*config.FlagStorageType)
+
+	// define custom type
+	type Input struct {
+		FlagValue bool `json:"value"`
+	}
+	var input Input
+
+	err := json.NewDecoder(r.Body).Decode(&input)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	storageType.SetBool(flagKey, input.FlagValue)
+
+	w.Header().Set("content-type", "application/json")
+	w.WriteHeader(http.StatusNoContent)
+	_, _ = w.Write([]byte(""))
+}
+
+func GetFlag(w http.ResponseWriter, r *http.Request) {
+	flagKey := r.PathValue("flagKey")
+
+	ctx := r.Context()
+	ctx_storage := ctx.Value(config.KeyVariable)
+	storageType := ctx_storage.(*config.FlagStorageType)
+
+	dbFlagKey, err := database.GetFlagKey(flagKey)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	validate := validator.New(ValidatorConfig)
+	var input payloads.GetFlag
+	d := json.NewDecoder(r.Body)
+	err = d.Decode(&input)
+
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	err = validate.Struct(input)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	var value any
+	var _err error
+	var contextualVariation datatypes.UUID
+
+	if input.Context.Attributes != nil && dbFlagKey.Enabled {
+		db := database.GetDB()
+		var flagRules []database.TargetingRule
+		result := db.Find(&flagRules, "flag_key_uuid = ?", dbFlagKey.UUID)
+		if result.RowsAffected > 0 {
+		outerLoop:
```

Comment:
Labels are a smell. See if you can get back to the label using functions instead. (the function can return instead of `break outerLoop`.)